app/lib/geordi-logger.coffee

```diff
@@ -4,12 +4,21 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
   constructor: (@state, @geordi) ->
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
+  @GEORDI_WARNINGS_TIMEOUT_INTERVAL = 5 * 60 * 1000 # only warn about Geordi down once every 5 mins, to avoid console spam
```

Comment:
no need for comments here the param is named well enough. Not sure about the caps though, is that consistent with the rest of the repo? Looks like @tokens doesn't use this convention.